apps/web/src/styles/account.css

```
@@ -483,6 +641,9 @@
   font-size: 12px;
   line-height: 1.6;
   color: #64748b;
+  min-width: 0;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)